Ginger/GingerCoreNET/Database/NoSqlBase/GingerCosmos.cs

```diff
@@ -125,9 +127,30 @@ public override void PerformDBAction()
                 {
                     case eDBValidationType.FreeSQL:
                         dbName = DatabaseName;
-                        containerName = SQLCalculated.Split(' ')[SQLCalculated.Split(' ').Length - 1];
+                        if (!SQLCalculated.Contains("where"))
```

Comment:
what if user will execute a select query without where clause ? code will go to else part